retroachievements/client.py

```diff
@@ -0,0 +1,127 @@
+import requests as request
+
+
+class RAClient:
+
+    # Header (required to pass cloudflare consistantly)
```

Comment:
Technically, this is not related to Cloudflare

Reply:
Cloudflare comment removed